### DISCUSSION TITLE
Hoe gaan we rekening houden met andere standaarden die al naar CB-NL mappen

### DISCUSSION THREAD

**Discussion by NielsHoffmann** [bimloket/CB-NL#16] [General]
bv UOB geeft aan dat ze al een mapping naar CB-NL hebben...
Als we straks de structuur van CB-NL aanpassen, kunnen zij hun mapping dan nog gebruiken?
Hoe gaan we ze helpen te 'migreren' naar de nieuwe structuur van CB-NL?

**Comment by hennystolwijk** [comment]
Ik zal hierover contact met de projectleider UOB opnemen. Want heeft gevolgen voor welke/hoeveel concepten we opnemen in de skos-laag.
